vlm/src/test/scala/geotrellis/contrib/vlm/GeoTiffRasterSourceSpec.scala

```diff
@@ -73,4 +73,88 @@ class GeoTiffRasterSourceSpec extends FunSpec with RasterMatchers with BetterRas
       assertRastersEqual(actual, expected)
     }
   }
+
+  it("should perform a tileToLayout") {
+    val targetCellSize: CellSize = CellSize(20.0, 20.0)
+
+    val testSource = GeoTiffReader.readMultiband(url).getClosestOverview(targetCellSize)
```

Comment:
There is a questions of a separate PR which i am working on, why the same test without selecting on overview works for GDAL?